src/actions/trackerActions.js

```diff
@@ -14,9 +14,13 @@ import type { Dispatch, GetState } from '../types/reduxTypes'
 
 import { PREHISTORIC_DATE, MODES, FOCUSES } from '../consts'
 
-export const createPoint = (description: string, timestamp: string) => ({
+export const createPoint = (
+  description: string,
+  timestamp: string,
+  issue?: ?string
```

Comment:
to be better backward compatible, we want to stay away from setting undefined as the issue. Hence issue is either supplied, or it's null. 
```suggestion
  issue?: ?string = null
```